packages/core/src/Menu/MenuContentImpl.vue

```diff
@@ -197,7 +198,7 @@ function handleKeyDown(event: KeyboardEvent) {
     // menus should not be navigated using tab key so we prevent it
     if (event.key === 'Tab')
       event.preventDefault()
-    if (!isModifierKey && isCharacterKey)
+    if (!isModifierKey && isCharacterKey && !isKeyDownInTextField)
```

Comment:
This change doesn't seem to be related to the issue the PR is trying to resolve. Could it be separated into another issue (with description) and PR? 🤔